stacks/dlaas/mkl/README.md

```diff
@@ -0,0 +1,25 @@
+# Clear Linux Deep Learning Stack
+[![](https://images.microbadger.com/badges/image/clearlinux/dlaas-mkl.svg)](http://microbadger.com/images/clearlinux/dlaas-mkl "Get your own image badge on microbadger.com")
+
+This provides a Clear Linux Deep Learning Stack. To offer more flexibility, there are two versions of the Deep Learning Stack: Tensorflow* framework optimized for Intel Architecture and a second version that includes TensorFlow* framework optimized using Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN) primitives.
+
```

Comment:
latest nomenclature is the `Eigen` version for Tensorflow and the `Intel MKL-DNN` version for mkl.